Tiltfile

```diff
@@ -220,34 +220,28 @@ def create_crs():
     local("kubectl apply -f templates/addons/calico-resource-set.yaml")
     local("kubectl apply -f templates/addons/flannel-resource-set.yaml")
 
-# run worker clusters specified from 'tilt up' or in 'tilt_config.json'
+# create flavor resources from cluster-template files in the templates directory 
 def flavors():
-    config.define_string_list("templates-to-run", args=True)
-    config.define_string_list("worker-flavors")
-    cfg = config.parse()
-    worker_templates = cfg.get('templates-to-run', [])
-   
     substitutions = settings.get("kustomize_substitutions", {})
     ssh_pub_key_B64 = "AZURE_SSH_PUBLIC_KEY_B64"
     ssh_pub_key_path = "$HOME/.ssh/id_rsa.pub"
     if substitutions.get(ssh_pub_key_B64):
         os.environ.update({ssh_pub_key_B64: substitutions.get(ssh_pub_key_B64)})
     else:
-        print("{} was not specified in tilt_config.json, attempting to load {}".format(ssh_pub_key_B64, ssh_pub_key_path))
+        warn("{} was not specified in tilt-settings.json, attempting to load {}".format(ssh_pub_key_B64, ssh_pub_key_path))
         os.environ.update({ssh_pub_key_B64: base64_encode_file(ssh_pub_key_path)})
 
     ssh_pub_key = "AZURE_SSH_PUBLIC_KEY"
     if substitutions.get(ssh_pub_key):
         os.environ.update({ssh_pub_key: substitutions.get(ssh_pub_key)})
     else:
-        print("{} was not specified in tilt_config.json, attempting to load {}".format(ssh_pub_key_B64, ssh_pub_key_path))
+        warn("{} was not specified in tilt-settings.json, attempting to load {}".format(ssh_pub_key_B64, ssh_pub_key_path))
```

Comment:
do we really want to warn here? I don't know if it's worth making the UI yellow when defaulting to the ssh key file, @mboersma what do you think?

Reply:
I decided to add it as a warning because if the user made a mistake in setting the Azure key or the Azure base 64 key, the code could fall through to the default key file without them noticing. For example, if they made a typo in "AZURE_SSH_PUBLIC_KEY," the variable would not be set.

Reply:
If we're just pursuing a fallback strategy, I don't think it's worth issuing a warning here. The yellow warnings in tilt feel serious to me.